src/karts/abstract_kart.hpp

```diff
@@ -307,8 +312,8 @@ class AbstractKart : public Moveable
      *  \param fade_out_time How long the maximum speed will fade out linearly.
      */
     virtual void increaseMaxSpeed(unsigned int category, float add_speed,
-                                  float engine_force, int duration,
```

Comment:
Do NOT replace ticks with floats. Lots of work was done to switch from imprecise floats to deterministic ints, please do not undo that